disallowSpaceAfterComma: New option

Francisc opened the pull request:
Disallows spaces after commas, for example `[1, 2, 3]` is invalid and
`[1,2,3]` is valid.

Fixes #1889

hzoo commented:
I think this should of been asked in the other rules - but can you add a test for new lines between commas instead of a space? We could also have a comment in between. Otherwise looks good

Francisc commented:
OK, I'll do that now.

Francisc commented:
Newline char is allowed by this rule.
Which makes sense since I'm looking to disallow the space and tab chars.
Do you want me to add a test that validates that newline characters are OK?

hzoo commented:
Yeah just checks that it doesn't error or something

Commit:
disallowSpaceAfterComma: New option

Disallows spaces after commas, for example `[1, 2, 3]` is invalid and
`[1,2,3]` is valid.

Fixes #1889

Commit:
Fix: Improved tests for disallowSpaceBeforeComma.

Francisc commented:
Hope this is better.
I've also updated the tests for `disallowSpaceBeforeComma`.

hzoo commented:
Yeah we just want to cover as many cases as we can think of to catch any potential bugs - looks good. I'm probably going to wait on merging new rules but this should be part of 2.4.0 which we should release before next monday at least.

Francisc commented:
Alright, thanks.

hzoo commented:
Check out our https://github.com/jscs-dev/node-jscs/blob/master/CONTRIBUTING.md and other rules if you want to understand how JSCS works - it's usually pretty simple to add rules. I usually use astexplorer.net to check the code's AST

Francisc commented:
I did read the CONTRIBUTING.md file! :)
Thanks for the http://astexplorer.net, didn't know about it.
Should come in handy.

hzoo commented:
Thanks again!
